docs/setup.md

```diff
@@ -23,10 +23,9 @@ Here are instructions for setting up a Python environment using Miniconda:
    conda create -n infra -c conda-forge python=3.10 poetry
    ```
 
-   The following pronpt will appear, "_The following NEW packages will be INSTALLED:_ "
-   You'll have the option to accept or reject by typing _y_ or _n_. Type _y_
-3.
-4. Activate the `infra` environment:
+   The following prompt will appear, "_The following NEW packages will be INSTALLED:_ "
+   You'll have the option to accept or reject by typing _y_ or _n_. Type _y_ to continue.
+   3. Activate the `infra` environment:
```

Comment:
Ah, I see what's happening here: the `3.` needs to be dedented to be at the same level as the other numbers

Reply:
So I tried that and it still stays on the same line (right side is from my LocalHost version: 

![image](https://github.com/cagov/data-infrastructure/assets/8131343/37e29dae-77f9-4007-82cb-e68417c1b14f)

If I do a double-space at the end of the previous sentence it does put it on a new line (but not indented correctly with the other numbers). However when it goes through checks the double-space gets removed. 

![image](https://github.com/cagov/data-infrastructure/assets/8131343/e106c387-73fc-489a-aa6a-a9939ddf6b28)

Reply:
There was an extra newline which was breaking the markdown list (in addition to the dedenting). I pushed a commit to this branch fixing it